test/DaySpec.hs

```diff
@@ -20,7 +30,7 @@ spec = parallel $ describe "Day" $ do
       it "returns 5" $ zellers 1  1  2000 `shouldBe` 5
       it "returns 6" $ zellers 28 12 2999 `shouldBe` 6
 
-    context "Sad Path" $ do
+    context "Sad Path" $
```

Comment:
Haskell uses `do` as [syntactic sugar](https://en.wikipedia.org/wiki/Syntactic_sugar) for Monadic bind (`>>=`). You'll probably want to check out the Monad chapter in the Haskell Book for a detailed explanation. [Here](https://en.wikibooks.org/wiki/Haskell/do_notation) are [two](http://www.haskellforall.com/2014/10/how-to-desugar-haskell-code.html) other resources.